pkg/mtls/testutils/testdata/central-certs/ca-key.pem

```diff
@@ -1,28 +1,52 @@
 -----BEGIN PRIVATE KEY-----
-MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDCKdA83BIKblVS
-3Q3777CABIN/19JIdck4+muJLz1MbRPCtdnA/YATVYnmd2RCpdMe03p6BFOrV+5z
-Ygia0Krbb17+Dxx1Qa798nCnQvIeV2CobPytxXRbkoLCa+jvffXsX0/4hNSLdDYT
-OT/aJlcGOOK2/F4s6rYYAeAZJrx605gpCcf5XNGn2Qd0DILeIe3OHKgsPqomj0lg
-Q3UWtXDREMxC/oHl+YIb3Lri+I5yS/aRdHlukIfoSFEoPADFGmnLCROEE169Oy1Z
-v2sU9khbDo/gihznoFazdiWd7nkwpdWxTD6D7NAmdpe/1dwzujOTV6caua87a18/
-YEm3cbAHAgMBAAECggEAWZ4FEGIfLZNZbIK9VqB+Hch3jYVm+Kzs8JCdG9ei+miK
-K1/WHv6io8nieW535080j9DaggEHQVm3pKgGArb3dYsFFEyTPQfRJkNFfd4ndJut
-8G9FfUta9n08P1KhVBEWdkBT4AgnNiqwXtUYxK7XkfalpgH70c2PuKr87X70ldfP
-tIj9OLQsXxzKgaVssdpjloVb4e9Gr1bvw1HjxARlj1a/Y5/X03xyXixojGQFiQlv
-oqmUmiIXXxQFrEKNBKsQ/kjnYdIGihY224V4YgJwVJparTHy9nDY5w5XYD0G027O
-hAto/NwZ9kMjaoya9vO9a7JhkLkYMFzJvVwh/6lBwQKBgQDxoxHU28FilCNEl1mp
-L/qL9D753dCfua4+0weLjz4XXVdIozhgWg1dA9raQaKZ1FJE6e3w4fRuQeYlnV5S
-CEKLwvMpNUpWHJ35KMIkD/dreS8fiiucMYIqGQ1pnmMhK335pVEsRKWoRgJOIEPd
-Fok7Xpdl6bi9ThaIhoYc0WD9pwKBgQDNtFlPpSyO7f0XYRiF4of8Ex6DYQwb6/j1
-AKPA7I5y4sh/AjKdlli+l5CeSALxFDr4+NZJjWzA3uWLc8bYx8szcCg3HNUqrpJh
-Sxy1zBuxlbUpz09TvmNOaf1eMmdxXyWXhb1mhTUsV5rw9UrxzipbUnMcgvI6fFlu
-8B4HhwXGoQKBgQCfPyAGT1mwlP51OuXNOaB+UdabJ5NRYLa0ErG7SQjbrxlR9re1
-0ENUVLEL/AukI6igYCb/BHrpmyaJDaBWobjxkOZd9w/plmyKA1GHILqnqrLir1W0
-V3B4PBB51fdwqdAZEdlgOlGOh60HPf5bm3mPlfRjJtgocmkpSrbiER9BqwKBgAQG
-6tSvVB78M784bGV6mNTrfuZtiPIm/Xm2aH5HY0aGgAxUe68wGU6p4u/mUwBrN/hV
-zsQj26Qu+Olw+8yUpLq+EFt72gR65Gy5Lre0ZjpkxuYpo9Ds4j7PvmEQtYu3pCHC
-QYH3S6IxmNvswWnYcmXVSGGfgqR4eLu+Qag+EXPhAoGAQj89wNgQ4WOZHnJOEORN
-bD7W9bmzBLJGWi8SiQs95ofRLrtS8KUiJuNlVFt0Cczqmmv02ynlPSlr522mHBj/
-QTjv5oXXO0pWZc84oq5kHgvoomu3FxEeuSreVKsQs1QiEhsjecStmTZeplehPkMf
-ty+nEkCChJ3Kj8p37JVPmKY=
+MIIJQgIBADANBgkqhkiG9w0BAQEFAASCCSwwggkoAgEAAoICAQC/Ibsc33Rcg1r3
```

Comment:
These and the following files needed to be changed because go1.18+ will refuse to validate SHA-1 signatures. The generation logic has been adjusted to pass `-sha256` to `openssl x509` in order to fix this.